zuul.d/nodesets.yaml

```diff
@@ -35,12 +35,12 @@
     nodes:
       - name: centos-8
         label: centos-8-1vcpu
-      - name: vmware-vcsa-6.7.0
+      - name: vcenter
```

Comment:
I still think it is helpful to have version number listed in console logs or inventory, but we can loop back once we add a 2nd version.

Reply:
yes, but no. Sorry. The vcenter hostname during the test is `vcenter.test` anyway, and we cannot change it. In addition, I want the whole automation to be agnostic as possible.